static/sass/_snapcraft_tour.scss

```diff
@@ -1,5 +1,5 @@
 @mixin snapcraft-tour {
-  $color-overlay: transparentize($color-dark, .15);
+  $color-overlay: transparentize($color-dark, .35);
```

Comment:
Did you chat to anyone about this? I'd be tempted to go even further to like `.65` or `.75`

Reply:
I talked to @spencerbygraves and he said that we are not actually to change this (as it's vanilla), but if we change it the change should subtle and he recommended this.